docs/supplement.rst

```diff
@@ -7,24 +7,39 @@ Guidelines and Procedure to Update the Supplement
 
 Currently, the guidelines for adding bad stars to the supplement are spelled out in
 `the wiki page of the agasc module Github repository
-<https://github.com/sot/agasc/wiki/Add-bad-star-to-AGASC-supplement-manually>`_.
+<https://github.com/sot/agasc/wiki/Add-bad-star-to-AGASC-supplement-manually>`_. They need to be
+modified to agree with this documentation.
+
+The supplement file is regularly updated as part of the Aspect group operations. The update and
+promotion process follows these steps:
+
+- Candidate supplement update. The table of magnitudes in the supplement is updated. Usually on a 
+  Sunday. A series of reports are produced, including a report of "suspicious" observations, which
+  require inspection. Reports are placed in
+  `agasc/supplement_reports <https://cxc.cfa.harvard.edu/mta/ASPECT/agasc/supplement_reports>`_
+- Report review. The aspect group inspects the reports as part of the weekly meeting. Upon reviewing
+  the reports, a decisions is made on whether to:
+
+  - exclude/include specific star observations from the magnitude estimation.
+  - add a bad star
+  - set a star magnitude to a given value
+
+- Observation disposition. The aspect reviewer decides on the status of suspicious observations
+  (and bad stars), edits $SKA/data/agasc/obs-status.yml accordingly,
```

Comment:
It seems like this file should live in `$SKA/data/agasc/rc/`, not in the flight area.

Reply:
Right. I will update it together with other docs changes (note that I have edited the description of this PR after I created it, and that text should match the rst).

Reply:
I made this change to the PR description, so I will add it with the others to the rst.

Feel free to change the PR description where it describes the workflow.